core/src/main/scala/kafka/admin/ConfigCommand.scala

```diff
@@ -278,71 +275,90 @@ object ConfigCommand extends Config {
 
   class ConfigCommandOptions(args: Array[String]) {
     val parser = new OptionParser(false)
-    val zkConnectOpt = parser.accepts("zookeeper", "REQUIRED: The connection string for the zookeeper connection in the form host:port. " +
+    val zkConnectOpt = parser.accepts("zookeeper", "The connection string for the zookeeper connection in the form host:port. " +
             "Multiple URLS can be given to allow fail-over.")
             .withRequiredArg
-            .describedAs("urls")
+            .describedAs("url(s) for the zookeeper connection")
             .ofType(classOf[String])
-    val alterOpt = parser.accepts("alter", "Alter the configuration for the entity.")
-    val describeOpt = parser.accepts("describe", "List configs for the given entity.")
-    val entityType = parser.accepts("entity-type", "Type of entity (topics/clients/users/brokers)")
+            .required
+    val alterOpt = parser.accepts("alter", "Alter the configuration for the entity. This is required unless the user specifies --describe option. The " +
+                                  "entity type must be specified using option --entity-type. " +
```

Comment:
Vahid, this is to address your comment about adding requiredUnless for alter and describe options since either one of those action was required. 

RequiredUnless can only be applied to one of the options: alter or describe and not both at the same time. If I apply it to on one of them, say for example we have 2 options : A and B and we say: B is requiredUnless A and in the command line we do not mention A or B, it will print an error message saying B is required. Ideally, we would want it to print 'A or B is required" and it does not look like JOptParser has a way to do this. 

So what I ended up doing is not use requiredUnless but instead in the description for alter and describe options, I mentioned that those options are required unless the other is specified. Hope that helps.